### ISSUE TITLE
Ban KingsRock in Wz

### ISSUE THREAD

**Issue by Firemelon66** [SmashMC-Development/Suggestions#1000]
Imo Kingsrock should be banned in wz how it was last reset everyone who Plays a bit of wz should know why and even in the competitive tiers on showdown it is banned
**Which servers does this apply to?All

**Comment by morrow12** [comment]
yeah I agree with this kings rock is very easily spammed on pokemon who hit multiple times for example cloyster always hits 5 so it's chance to flinch is like 50% this item is even banned in Ubers that's how op it is.

**Comment by Sadkeemstar112** [comment]
Kings rock is still legal in AG people should not get restricted to What items they can use in warzone and its only like 3 mon that i can think of that really can use Kingsrock well. So i dont agree with this and kingsrock maybe can be rng based but its not worthy of a ban in warzone

**Comment by Firemelon66** [comment]
But then they should make it full ag again like add evasion and baton pass again and dont ban moody and stuff

**Comment by WalkAway7** [comment]
> But then they should make it full ag again like add evasion and baton pass again and dont ban moody and stuff

I think it's a difference with that, since with evasion and moody, its just easy to setup and baton pass into a sweeper and just sweep everything, which is just stupid, Kings rock on the other hand is more rng based. So if we ban Kings Rock, ranchi + iron head should be banned too

**Comment by morrow12** [comment]
not really lol, rachi can get countered unlike cloyster since it hits 5 times with 50 chance flinch kings rock is not balanced for the meta.
